packages/shared/tests/utils/wait.test.ts

```diff
@@ -7,7 +7,7 @@ describe('wait function', () => {
 
     const waitPromise = wait(delay).then(mockFunc);
 
-    jest.advanceTimersByTime(delay);
+    jest.useFakeTimers();
```

Comment:
This is in the wrong spot and it actually looks like your tests aren't following the patterns from the docs; please revisit the docs and update as needed: https://jestjs.io/docs/timer-mocks